### ISSUE TITLE
Would it be possible to expose the Config type?

### ISSUE THREAD

**Issue by y-hsgw** [scaffdog/scaffdog#1267]
## Expected Behavior

Hi 🖐️
Would it be possible to expose the [Config type](https://github.com/scaffdog/scaffdog/blob/canary/packages/%40scaffdog/types/src/config.ts#L6) from @scaffdog/types?
I would like to perform type-checking when creating a config.ts file as shown below:

config.ts
```ts
import { type Config } from "scaffdog";

const config: Config = {
  files: ['*'],
};

export default config;
```

If there is an alternative approach, I would appreciate your guidance 🙇‍♂️

## Your Environment

- OS:  macOS Sonoma
- scaffdog version: 4.0.0

**Comment by wadackel** [comment]
Hi @y-hsgw, Thank you for using scaffdog :feet:

I believe you can import the Config type by installing `@scaffdog/types` :+1:
Do you want to import the Config type directly from the `scaffdog` package?

Your feedback makes scaffdog even better!

**Comment by y-hsgw** [comment]
Thank you for your response😄

> Do you want to import the Config type directly from the scaffdog package?

Yes, I think it would be great if I could import it directly, as it would also eliminate the need to manage an additional package (`@scaffdog/types`)!

**Comment by wadackel** [comment]
Thank you for your valuable feedback! I plan to update the `scaffdog` package to export the `Config` type :+1:

**Comment by wadackel** [comment]
Released as [v4.1.0](https://github.com/scaffdog/scaffdog/releases/tag/scaffdog%404.1.0). Please check it out if you have time :dog:

**Comment by y-hsgw** [comment]
Thank you for your prompt response!
I wish you all the best in your future development🐕
